redisinsight/api/src/modules/cloud/session/repositories/local.cloud.session.repository.ts

```diff
@@ -0,0 +1,43 @@
+import { InjectRepository } from '@nestjs/typeorm';
+import { Repository } from 'typeorm';
+import { classToClass } from 'src/utils';
+// import { SessionMetadata } from 'src/common/models';
+import { EncryptionService } from 'src/modules/encryption/encryption.service';
+import { ModelEncryptor } from 'src/modules/encryption/model.encryptor';
+import { CloudSessionRepository } from './cloud.session.repository';
+import { CloudSessionEntity } from '../entities/cloud.session.entity';
+import { CloudSessionData } from '../models/cloud-session';
+
+const SESSION_ID = 1;
+
+export class LocalCloudSessionRepository extends CloudSessionRepository {
+  private readonly modelEncryptor: ModelEncryptor;
+
+  constructor(
+    @InjectRepository(CloudSessionEntity)
+    private readonly repository: Repository<CloudSessionEntity>,
+    private readonly encryptionService: EncryptionService,
+  ) {
+    super();
+    this.modelEncryptor = new ModelEncryptor(this.encryptionService, ['data']);
+  }
+
+  async get(): Promise<CloudSessionData> {
+    const entity = await this.repository.findOneBy({ id: SESSION_ID });
+
+    if (!entity) {
+      return null;
+    }
+
+    const decrypted = await this.modelEncryptor.decryptEntity(entity, false);
+    return classToClass(CloudSessionData, decrypted);
+  }
+
+  async save(cloudAuth: Partial<CloudSessionData>): Promise<void> {
+    const entity = await this.modelEncryptor.encryptEntity(
+      classToClass(CloudSessionEntity, { id: SESSION_ID, ...cloudAuth }),
```

Comment:
I would like to rewrite this to `{  ...cloudAuth, id: SESSION_ID }` to make sure that id will be hardcoded